backend/scripts/ci/lint.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -uxo pipefail
+
+# Python Code Quality Checks
+
+# return 0 if all checks return 0
+# 1 instead
+
+pylint --rcfile=.pylintrc ${@:-core tournesol}
+chk1=$?
+
+flake8 --config=.flake8 ${@:-}
+chk2=$?
+
+exit $chk1 && $chk2
```

Comment:
This does not seem to be correct. The exit code is always `$chk1`.

According to https://stackoverflow.com/a/16358989/5634396 a possible solution is
```suggestion
! (( chk1 || chk2 ))
```
```

Reply:
My bad, it's indeed not correct.